Emby.Dlna/Didl/DidlBuilder.cs

```diff
@@ -782,36 +782,33 @@ private void WriteObjectClass(XmlWriter writer, BaseItem item, StubType? stubTyp
 
         private void AddPeople(BaseItem item, XmlWriter writer)
         {
+            if (!item.SupportsPeople)
+            {
+                return;
+            }
+
             var types = new[]
             {
                 PersonType.Director,
                 PersonType.Writer,
                 PersonType.Producer,
                 PersonType.Composer,
-                "Creator"
+                "creator"
             };
 
-            var people = _libraryManager.GetPeople(item);
-
-            var index = 0;
-
-            // Seeing some LG models locking up due content with large lists of people
-            // The actual issue might just be due to processing a more metadata than it can handle
-            var limit = 6;
+            var people = _libraryManager.GetPeople(
+                new InternalPeopleQuery
+                {
+                    ItemId = item.Id,
+                    Limit = 6
```

Comment:
I would suggest to keep the comment explaining why we limit at 6. DLNA is a black magic thing, so keeping all knowledge we have is needed

Reply:
Done